deployments/helm/values-dev.yaml

```diff
@@ -80,11 +80,11 @@ ingress:
 
 resources:
   requests:
-    cpu: 800m
-    memory: 2Gi
+    cpu: 20m
+    memory: 30Mi
   limits:
-    cpu: 800m
-    memory: 2Gi
+    cpu: 40m
+    memory: 50Mi
```

Comment:
@sanjay-ba I once observed in the `saas-dev` or `saas-prod` cluster the DE is crashing when such low values are set (I did remember the exact values). Have you tested whether the DE is able to generate policies with these values/limits configured?